installKubeVIP

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -eu
+export VIP_IP_ADDRESS=$1
+export VIP_INTERFACE=$2
+export TAG=v0.6.4
+
+# check that the interface exists
+ip r | grep default | grep $VIP_INTERFACE || {
+	echo interface $VIP_INTERFACE not found on default route - aborting
+        exit 1
+}
+
+# Get the RBAC config from the website
+mkdir -p /var/lib/rancher/rke2/server/manifests/
+curl -s https://kube-vip.io/manifests/rbac.yaml > /var/lib/rancher/rke2/server/manifests/kube-vip-rbac.yaml
+
+# use podman to create the VIP service via addon
```

Comment:
Based on https://kube-vip.io/docs/installation/daemonset/, we can use containerd (`ctr`) which is available to us by default when installing RKE2. This avoids installing and using podman.

Reply:
Yeah - spot on. I didn't look to use the ctr within the rancher install and did the expedient / hack of installing podman. I've redone the process with `ctr` and will amend by branch / PR later.